From 0985b32279d47e7ea01152233d26eedcdf721d09 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Fri, 11 Oct 2024 14:24:51 +0100
Subject: [PATCH 1319/1350] dts: bcm2712-rpi-500: Add USER_LED GPIO name

The USER_LED signal was missing from the initial DTS.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm64/boot/dts/broadcom/bcm2712-rpi-500.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm64/boot/dts/broadcom/bcm2712-rpi-500.dts
+++ b/arch/arm64/boot/dts/broadcom/bcm2712-rpi-500.dts
@@ -35,7 +35,7 @@
 		"PWR_GPIO", // GPIO_020
 		"KEYB_RUN", // GPIO_021
 		"-", // GPIO_022
-		"-", // GPIO_023
+		"USER_LED", // GPIO_023
 		"BT_RTS", // GPIO_024
 		"BT_CTS", // GPIO_025
 		"BT_TXD", // GPIO_026
